src/scikit_build_core/settings/skbuild_model.py

```diff
@@ -309,6 +309,18 @@ class WheelSettings:
     may not respect this exclusion.
     """
 
+    exclude_exts: List[str] = dataclasses.field(
+        default_factory=lambda: [".pyc", ".pyo"],
+        metadata=SettingsFieldMetadata(display_default="true"),
+    )
+    """
+    A set of file extensions to exclude from the wheel.
+
+    This is additive to the SDist/Wheel exclude patterns. This applies to the final paths
+    in the wheel, and can exclude files from CMake output as well.  Editable installs
+    may not respect this exclusion.
```

Comment:
Please add a `:::{versionadded} 0.12` in here and any other sphinx relevant parts you see fit.